apps/api_accounts/mix.exs

```diff
@@ -56,7 +56,8 @@ defmodule ApiAccounts.Mixfile do
      {:comeonin, "~> 5.1"},
      {:bcrypt_elixir, "~> 2.0"},
      {:uuid, "~> 1.1"},
-     {:bamboo, github: "mbta/bamboo", ref: "6ae35de"}]
+     {:bamboo, github: "mbta/bamboo", ref: "6ae35de", override: true},
```

Comment:
@paulswartz I had to set `override: true` here because this conflicts with the dependencies of `bamboo_ses`. Does this seem like the right approach?

Reply:
what's in our fork of `bamboo`? I don't recall what we changed.

Reply:
https://github.com/thoughtbot/bamboo/compare/master...mbta:master

Reply:
That seems like we can move back to whatever the hex-released version of Bamboo is.

Reply:
wil do